### DISCUSSION TITLE
How to change splash screen in grub?

### DISCUSSION THREAD

**Discussion by Topbanaana** [SpiralLinux/SpiralLinux-project#233] [Q&A] [answered]
Tried to do in my Budgie distro

export GRUB_MENU_PICTURE=/home/top/image.jpg

but it does not do the trick. Nothig changed (not that it was not beautiful that rainy green splash, but I am just curious).

Furthermore it looks I have on my ASUS Vivobook following queue:

1. ASUS splash screen
2. Grub menu
3. ASUS splash screen
4. Debian logo (excellent looks marvelous)
5. Old wallpaper I used some time ago for a moment
6. Lightdm Login  with new wallpaper I installed recently
7. After login new wallpaper refreshed

Once more, not that I object anything, it is fine now but this queue is a bit strange as compared to other distributions. Is this a Budgie special thing?

**Comment by artytux** [comment, marked as answer]
@Topbanaana

While I don't use Budgie
KDE suits my workflow better
I'm sure you will find the path on how to do this for budgie 

I changed the splash screen doing this in KDE

go in as root, into ~ boot > grub > found the image for splash screen, no surprise named "splash.png"
changed over to new image keep same name "splash.png"
dimensions if bigger no problem  kept values to screen proportions ratio   and  Mb  don't go too big
get out of root mode
Has worked for months no hassles
done
reboot Done
